manifest.json

```diff
@@ -1,8 +1,27 @@
 {
     "manifest_version": 2,
     "name": "Totally for real actual ghostery extension",
-    "version":"0.000001",
-    "content_scripts": [
+    "version":"0.0.1",
+    "permissions": [
+        "webRequest",
+        "http://*/*",
+        "https://*/*",
+        "webNavigation",
+        "cookies",
+        "tabs",
+        "ws://*/*",
+        "wss://*/*",
+        "storage"
```

Comment:
We don't actually need any of these quite yet, but we will later, so feel free to leave them in there. Just be mindful of the fact that only the first three entries are necessary for what we are doing right now.